appendix-basic65.tex

```diff
@@ -220,168 +252,240 @@ \subsection{BASIC 65 Variables}
-programming dialects. The usage and precedence of these operators is documented in this section.
-
-The \screentext{=} symbol is used both as an assignment operator, and as a relational
-operator for testing equality. For example, in the statement
-\screentext{A = B = 5}, the first equal sign is the assignment operator,
-while the second is a logical operator, comparing the variable \screentext{B} with \screentext{5}.
-The value of \screentext{A} will either be assigned the value \screentext{-1} (for {\bf TRUE}), or
-the value \screentext{0} (for {\bf FALSE}).
-You may have noticed that the value of \screentext{-1} for {\bf TRUE} is different to
-other programming languages, such as {\bf C}, where the value of \screentext{1}
-is used for {\bf TRUE} instead.
-
-The \screentext{+} symbol can be used as a positive sign for numerical expressions,
-as an addition operator, or for string concatenation. The number and type of operands
-determines the operation.
-
-The \screentext{-} symbol can be used as a negative sign for numerical expressions,
-or as a subtraction operator. The number and type of operands
-determines the operation.
-
-The operators \screentext{NOT, AND, OR} and \screentext{XOR} can be used both as
-logical operators, or as boolean operators.
-
-\begin{itemize}
-  \item Logical Operator Example: \screentext{IF A>B AND A<0}
-  \item Boolean Operator Example: \screentext{A = B AND \$7F}
-\end{itemize}
-
-Both examples always produce an integer result internally, which can be interpreted
-either numerically or logically.
-If the result of a comparison is {\bf TRUE}, the value will be set
-to {\bf -1}, while a {\bf FALSE} result yields {\bf 0}.
-In the boolean operator example above, the \screentext{AND} operator converts both operands
-to a 16-bit integer value, and performs a bitwise {\bf AND} for all 16 bits.
-This example will take the value of \screentext{B}, set the upper 9 bits to zero,
-and store the result in \screentext{A}.
-
-The result of logical operations can be used in numerical expressions as well,
-for example,
-\screentext{A = A - (B > 7)}
-will increment \screentext{A} by 1 if the result of \screentext{(B > 7)} is {\bf TRUE} (-1). This is because
-the mathematical expression of \screentext{A = A - (-1)} is the same as \screentext{A = A + 1}.
-
-The operators have precedences, which are listed in the tables below.
-In the statement \screentext{A * A - B * B}
-both multiplications will be performed first, before the subtraction is executed.
-Parentheses are used to change the precedence, for example
-\screentext{A * (A - B) * B)} will execute the subtraction first.
-
-\subsection{Assignment Operator}
+An {\em operator} is a symbol or keyword that performs a function in an
+expression. It operates on one or two sub-expressions, called {\em operands}.
+The part of the expression that involves the operator is treated as the result
+of the operation: the operator {\em evaluates to} the result.
```

Comment:
I had to re-read this line a few times to try get the gist of it, and I still feel a bit blurry on it. Maybe the colon in the middle of it adds to the confusion, not getting what the colon wants to contribute to it.

Not sure what to suggest on this, maybe I'd just go for "The operator and its operands evaluate to a result to be used by the expression." But maybe I've missed something in what you wanted to convey, so you might have other thoughts on this

Reply:
Good note, I attempted to simplify.